### ISSUE TITLE
VSTs not loading on Linux with Wine

### ISSUE THREAD

**Issue by little512** [LMMS/lmms#5930]
### Bug Summary
Vestige fails to load any VSTs after recent system update
#### Steps to reproduce
* Open LMMS
* Load Vestige into a project
* Try to load any valid VST .dll
#### Expected behavior
The VST loads and is able to be used
#### Actual behavior
The VST fails to load with an error message
#### Affected LMMS versions
LMMS 1.2.2

#### Logs
<details>
  <summary>Click to expand</summary>
<pre>
*** WEAK-JACK: initializing
*** WEAK-JACK: OK. (0)
Notice: could not set realtime priority.
Connection established.

Stream successfully created
002c:fixme:winediag:LdrInitializeThunk wine-staging 6.3 is a testing version containing experimental patches.
002c:fixme:winediag:LdrInitializeThunk Please mention your exact version when filing bug reports on winehq.org.
0024:err:seh:segv_handler_early Got unexpected trap 14 during process initialization
remote plugin died! invalidating now.
Remote plugin did not connect.
</pre>
</details>

VSTs did work for me before a recent system update where wine-staging 6.3 and some of it's dependencies were updated. I tried installing wine-stable and I got the same issue. I also tried with a fresh wine prefix with no luck, which leads me to believe it has to do with one of wine's optional dependencies since that's the only thing that changed on my system.

**Comment by Sauerstoffdioxid** [comment]
Same here. Using the 1.2.2 AppImage. My distro (Solus, in case that matters) updated from wine 6.1 to 6.3 (non-staging, along with a bunch of other updates) and VSTs became unusable. When I launch LMMS from console, it just says the following and hangs for a bit.
```
remote plugin died! invalidating now.
Remote plugin did not connect.
```

**Comment by akimaze** [comment]
@little512 @Sauerstoffdioxid I think this is bug in latest wine (6.2 works OK). See https://bugs.winehq.org/show_bug.cgi?id=50731

**Comment by little512** [comment]
@akimaze It seems this bug has been fixed and only applies to applications built with Winelib using winegcc/g++, my VSTs don't fit into this category so I'm not sure this is relevant.

**Comment by akimaze** [comment]
@little512 I think `RemoteVstPlugin` crashes not your VST. When I downgrade to 6.2 everything works for me. You are right, it's fixed but not released yet - we must wait for 6.4 to test that :)

**Comment by little512** [comment]
Update: wine-staging 6.4-1 works with Vestige, closing the issue now.
